src/components/Reports/Router.tsx

```diff
@@ -19,7 +17,7 @@ const ReportsRouter = (): JSX.Element | null => {
   return (
     <Switch>
       <Route exact path={APP_PATHS.REPORTS}>
-        {featureFlagProjects ? <ReportsView tab={'reports'} /> : <ReportList />}
+        <ReportsView tab={'reports'} /> : <ReportList />
```

Comment:
remove ReportList?

Reply:
it is outside of an evaluation `{..}` and considered raw jsx.. the ':' would be considered a string maybe